ssl/statem/statem_clnt.c

```diff
@@ -594,6 +594,8 @@ unsigned long ossl_statem_client_max_message_size(SSL *s)
             return SERVER_HELLO_DONE_MAX_LENGTH;
 
         case TLS_ST_CR_CHANGE:
+            if (s->client_version == DTLS1_BAD_VER)
```

Comment:
Should that be s->version ? I've never quite understood the distinction, in client code.